package.json

```diff
@@ -48,10 +48,10 @@
   },
   "dependencies": {
     "arrify": "^2.0.1",
-    "fs-extra": "^9.0.1",
     "micromatch": "^4.0.2",
     "schema-utils": "^3.0.0",
-    "@types/eslint": "^7.2.0"
+    "@types/eslint": "^7.2.0",
+    "webpack-sources": "^2.0.0"
```

Comment:
Do not use webpack-source directly, you can get them from `compiler.webpack`

Reply:
`compiler.webpack` is undefined on webpack 4.